File: X/rust/XRPL_UKL1/soal3.rs
/*
Soal          : 3
Gambar        : A
Matrix kolom  : 6
*/

fn main() {
    let a = 5;
    let b = 3;
    let kolom = 6;
    let mut u = a;
    let mut s = b;
    println!("Deret Aritmatikanya adalah\n");
    for i in 0..kolom {
        for _ in 0..i + 1 {
            print!("{}\t", u);
            u += b;
            s += u;
        }
        println!();
    }
    let total = s - u;
    println!("Jumlah Deret Aritmatikanya adalah {}", total);
}

File: X/javascript/XRPL_UKL1/soal3.js
/*
Soal          : 3
Gambar        : A
Matrix kolom  : 6
*/

a = 5;
b = 3;
kolom = 6;
u = a;
s = b;
console.log("Deret aritmatikanya adalah");

for (i = 0; i < kolom; i++) {
    for (j = 0; j < i + 1; j++) {
        process.stdout.write(u + "\t");
        u += b;
        s += u;
    }
    console.log();
}
total = s - u;
console.log("Jumlah deret aritmatikanya adalah " + total);